RELEASE-NOTES.md

```diff
@@ -9,6 +9,7 @@
 - Pass the `tune` argument from `sample` when using `advi+adapt_diag_grad` (see issue [#3965](https://github.com/pymc-devs/pymc3/issues/3965), fixed by [#3979](https://github.com/pymc-devs/pymc3/pull/3979)).
 - Add simple test case for new coords and dims feature in `pm.Model` (see [#3977](https://github.com/pymc-devs/pymc3/pull/3977)). 
 - Require ArviZ >= 0.9.0 (see [#3977](https://github.com/pymc-devs/pymc3/pull/3977)).
+- Temporarily fixed issue [#3962](https://github.com/pymc-devs/pymc3/issues/3962) by making change in the `_random()` method of `GaussianRandomWalk` class, refer to PR [#3985].Further testing revealed a new issue which is being tracked [#4010](https://github.com/pymc-devs/pymc3/issues/4010) 
```

Comment:
I think that particular issue of sampling is fixed for good, no? It just revealed another problem.

Reply:
[No](https://github.com/pymc-devs/pymc3/pull/3985#discussion_r453156946), When some edge cases were tested it turned out that a deeper issue exists in the function `logp`. This PR is only a temporary fix which would be resolved permanently with the issue [#4010](https://github.com/pymc-devs/pymc3/issues/4010) fixed

Reply:
The way I understand, there are two related but separate issues: 1D sampling from the GRW did not work (#3962), and the logp for >1D GRWs is wrong (#4010). This PR fixes the first completely but doesn't address the second.

Reply:
Yeah that's my understanding too. I think the notion of temporary fix should be removed from the release notes

Reply:
Ok then I shall make the corresponding changes in Release notes. However, I am keeping the referene to 4010